packages/ios_platform_images/ios/Classes/IosPlatformImagesPlugin.swift

```diff
@@ -0,0 +1,73 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import Flutter
+import Foundation
+
+public final class IosPlatformImagesPlugin: NSObject, FlutterPlugin {
+
+  private let channel: MethodChannel
+
+  init(channel: MethodChannel) {
+    self.channel = channel
+  }
+
+  public static func register(with registrar: FlutterPluginRegistrar) {
+    let channel = FlutterMethodChannel(
+      name: "plugins.flutter.io/ios_platform_images",
+      binaryMessenger: registrar.messenger())
+
+    let instance = IosPlatformImagesPlugin(channel: channel)
+    registrar.addMethodCallDelegate(instance, channel: channel)
+  }
+
+  public func handle(_ call: FlutterMethodCall, result: @escaping FlutterResult) {
+    switch call.method {
+    case "loadImage":
+      loadImage(call, result)
+    case "resolveURL":
+      resolveURL(call, result)
+    default:
+      result(FlutterMethodNotImplemented)
+    }
+  }
+
+  private func loadImage(_ call: FlutterMethodCall, _ result: @escaping FlutterResult) {
+    if let name = call.arguments as? [String], let firstItem = name.first {
+      guard let image = flutterImageWithName(withName: firstItem) else {
```

Comment:
Why did you replace [a call to `[UIImage imageNamed:]` with a call to `flutterImageWithName`? Those aren't the same method, and aren't intended to do the same thing at all.

Reply:
I added it back as it was in the ObjC code